catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -191,16 +218,33 @@ public UpdateStorageResponse version(UpdateStorageRequest streamUpdateRequest,
                 .collect(Collectors.toList());
 
         if (updatedMetacards.isEmpty()) {
+            LOGGER.trace("No updated metacards applicable to versioning");
+            SecurityLogger.audit("Skipping versioning updated metacards with ids: "
```

Comment:
For just a single concat I left it since didn't know if it would make much of a difference. (though it would be interesting to find out, is the parameterized logging more efficient if its a single param or is concatting  it 🤔 ) 

We can't use the isTraceEnabled because this is a `SecurityLogger.audit`. Also the resulting string should just be a list of metacard ID's, which hopefully isn't too big.  The above trace log should be OK since it is relatively small.